### ISSUE TITLE
Please help me choose the right ontology during experiment creation

### ISSUE THREAD

**Issue by Steffengreiner** [qbicsoftware/data-manager-app#887]
### What are you missing?

(PCT Workshop Feedback)
Currently, while selecting a species, specimen and analyte i am shown the same value (e.g. protein) multiple times with only the linked ontology as an additional information. I'd like to have additional information which helps me understand which value to select. 

Additionally, there are duplicate values from the same ontology selectable for which i don't see a difference. 
![Screenshot 2024-11-12 at 18 11 44](https://github.com/user-attachments/assets/e4d91928-dd5c-4dd9-83f8-8e3ed4a51db4)

### Why do you need a solution?

I don't know which value to select, therefore i will randomly select an ontology which is not how this feature is intended. 

### Ideas

Multiple solutions are possible. Let me select the ontology from a list of ontologies with additional information or provide me with additional information per ontology entry in the dropdown menu

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Steffengreiner** [comment]
Having more information(e.g. Ontology Source name) in the dropdown about the ontology would be greatly helpful for users to make a decision

**Comment by Shraddha0903** [comment]
There are 2 UI suggestions for the identified issue:

1. To provide the ontology name instead of the URI for each item in the combobox.

![Image](https://github.com/user-attachments/assets/e8400c83-7690-464c-ac5a-48fe1f7c8d34)

2. To provide the ontology name and a tooltip for the name to better understand the ontology. 

![Image](https://github.com/user-attachments/assets/cfff6440-1441-46f9-8095-25893c825276)

**Comment by sven1103** [comment]
I believe this is ready to work on and not a big task.

@Shraddha0903 

1.) how do we handle the case, when there is no description available for an ontology term
2.) The OBO ID (e.g. `BTO:1515165`) is now not accessible at all anymore. Is that intended?

**Comment by Shraddha0903** [comment]
@sven1103 

1. For this case, we still show the tooltip with the text - **No description available.**
Keeps the interaction consistent and avoids confusion about the tooltip being missing/broken.
2. I don't understand, can you explain? which term is it for? where is it not accessible?

**Comment by Steffengreiner** [comment]
As @sven1103 already mentioned, while the UI draft is solid it needs to be investigated how we handle:

1.) No description provided by OLS Service:

<img width="1607" height="121" alt="Image" src="https://github.com/user-attachments/assets/951523e7-f309-4861-a1eb-84291510f2de" />

2.) Insanely Long description provided by OLS Service:

<img width="1549" height="695" alt="Image" src="https://github.com/user-attachments/assets/4c0c453a-7429-4748-b423-8214a06a238a" />

3.) Nonsensical description provided by OLS Service

<img width="1541" height="209" alt="Image" src="https://github.com/user-attachments/assets/d5c51fd2-464d-4707-9db1-8cd56759c1cb" />

Basically we need to know if the definition/description field can be trusted in the OLS API

www.ebi.ac.uk/ols4/defined-response-fields
